### PR TITLE
Create landing page

### PR DESCRIPTION
**Pull request by Tianchai**
image

image

### CODE REVIEW

File: static/css/landing_page.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+#header {
+  width: 100%;
+}
+@media (min-width: 325px) and (max-width: 399px) {
+  .social-section > .rocket {
+    display: inline-block;
+  }
+}
+/* sm [phone landscape]: 576-767px */
+@media (min-width: 576px) {
```
Comment (vtno) [inline static/css/landing_page.css:60]: We should remove unused CSS-rule.

File: templates/index.html [hunk 2]
```diff
@@ -1,130 +1,95 @@
...
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/custom.css" media="screen">
+        {% endif %}
+        {% endblock head_links %}
+        {% include '_includes/favicon_links.html' %}
+        {% block feed_links %}
+        {% include '_includes/feeds.html' %}
+        {% endblock feed_links %}
+        {% include '_includes/analytics.html' %}
+        <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/landing_page.css" media="screen">
+        <style>
```
Comment (vtno) [inline templates/index.html:56]: This landing page-specific rule should be moved to the CSS file. This template file will probably be reused for multiple pages when we start developing the whole site.

**Review by vtno — CHANGES_REQUESTED** [review]
Apart from some minor comments, we should really optimize that 1.4Mb background image.

File: templates/index.html (hunk 2 above)
Comment (jean) [reply templates/index.html:56]: I'm not 100% sure at this point, but I think `index.html` is only for the landing page (others are `page`, `article`, `categories`, etc.). The reused one is `base.html`. 
That said, I don't have a problem with consolidating rules as long as it's convenient to target rules to the frontpage (e.g. if all pages have a unique `class`).

**Review by jean — APPROVED** [review]
I'm going to merge this so that we can get a feel for the development process. 
We can tune things in later PRs.

**Pull request merged**